SPECS/maven-jdk17/maven-jdk17.spec

```diff
@@ -0,0 +1,247 @@
+%global debug_package %{nil}
```

Comment:
I think you can move this spec file to the regular maven folder.  You don’t need to separate

Reply:
See what we did for golang:
https://github.com/microsoft/CBL-Mariner/tree/2.0/SPECS/golang

Reply:
We can move the spec file to regular maven folder. But since the difference b/w golang and maven is NEW package name (SAME version), we need to add the same in cgmanifest as well as in licenses mapping..